### DISCUSSION TITLE
Please someone help me with the installation. Windows Server , Windows 11 ...

### DISCUSSION THREAD

**Discussion by diskmail34** [snipe/snipe-it#14760] [Ideas]
Please someone help me with the installation.

I tried to install it on my password (windows server) and saw SSL errors and dozens of files swapped in the videos. Then on ubuntu and centos. It did not happen on a separate standalone laptop. Now, how can I install and distribute Windows 10 in a virtual machine on a separate independent laptop? Please help with installation.

**Comment by neokleon** [comment]
https://youtu.be/joiUjMhDaOk?si=SSxNtr1LYIDFwxVz
